### DISCUSSION TITLE
What's the new features in the new version (1.0.0)

### DISCUSSION THREAD

**Discussion by FelipeBattistotti** [vercel/swr#1415] [Q&A]
What's the new features in the new version?

Comparing version 1.0.0 with version 0.5.7

**Comment by kayk1m** [comment]
You might want to read [this](https://github.com/vercel/swr/releases/tag/1.0.0).

**Comment by shuding** [comment]
Please take a look at [Announcing SWR 1.0](https://swr.vercel.app/blog/swr-v1) for details :)
